src/appData/constants/index.js

```diff
@@ -13,7 +13,8 @@ export const FUNDRAISER_CATEGORY = {
 };
 
 export const FUNDRAISER_STATUS = {
-  CREATED: "CREATED",
+  DRAFT: "DRAFT",
```

Comment:
what are we using instead of CREATED, is it DRAFT only, what would be the status once the user clicks on create button after filling in details